src/site/outline/outline.rst

```diff
@@ -94,23 +94,35 @@ Below is a list of topics in no particular order that may or may not be covered
 Student Evaluation (Tentative Dates)
 ====================================
 
-+------------------------+------------+---------------------+
-|                        | Worth      | Due                 |
-+========================+============+=====================+
-| Assignment 1           | 5%         | Late January-ish    |
-+------------------------+------------+---------------------+
-| Assignment 2           | 5%         | Mid February-ish    |
-+------------------------+------------+---------------------+
-| Assignment 3           | 5%         | Mid March-ish       |
-+------------------------+------------+---------------------+
-| Assignment 4           | 5%         | Early April-ish     |
-+------------------------+------------+---------------------+
-| Test 1                 | 20%        | Late January-ish    |
-+------------------------+------------+---------------------+
-| Test 2                 | 20%        | Late February-ish   |
-+------------------------+------------+---------------------+
-| Final Exam             | 40%        | TBA                 |
-+------------------------+------------+---------------------+
+.. list-table:: Final Grade Breakdown
+    :widths: 50 50 50
+    :header-rows: 1
+
+    * - Deliverable
+      - Percentage
+      - Date/Due Date
```

Comment:
This is a confusing header imo. What does _date_ mean that _due date_ doesn't? When it's assigned? Why would that be the same value as the _due date_?  
_Due Date_ is already unambigious.

Reply:
The idea was that the test wasn't so much _due_ at that time, but I suppose when thinking about it more, it also is _due_ at that time. 

In other words, good point.